content/en/monitors/monitor_types/forecasts.md

```diff
@@ -9,39 +9,42 @@ further_reading:
   text: "Configure your monitor notifications"
 - link: "monitors/downtimes"
   tag: "Documentation"
-  text: "Schedule a downtime to mute a monitor"
+  text: "Schedule downtime to mute a monitor"
 - link: "monitors/monitor_status"
   tag: "Documentation"
   text: "Consult your monitor status"
 ---
 
-Forecasting is an algorithmic feature that allows you to predict where
-a metric is heading in the future. It is well-suited for metrics with
-strong trends or recurring patterns.
+Forecasting is an algorithmic feature that allows you to predict where a metric is heading in the future. It is well-suited for metrics with strong trends or recurring patterns.
 
 For example, if your application starts logging at a faster rate, forecasts can alert you a week before your disk fills up, giving you adequate time to update your log rotation policy. Or, you can forecast business metrics, such as user sign-ups, to track progress against your quarterly targets.
 
+To use forecast monitoring, start by creating a [forecast timeseries graph](#forecast-dashboard), and then [set up a monitor](#forecast-monitoring) to trigger when a metric is forecasted to hit a specific threshold.
```

Comment:
I wouldn't recommend this flow to users. The monitor creation page is designed to be self-contained. A user should be able to use the preview graphs on that page to understand what the metric looks like and what we'll forecast.

Reply:
That makes sense - I just completely removed it.